src/transform.ts

```diff
@@ -110,20 +111,17 @@ export interface WindowFieldDef {
   op: AggregateOp | WindowOnlyOp;
 
   /**
-   *  Parameter values for the window functions. Parameter value can be null for operations that do not accept a
-   *  parameter.
+   *  Parameter values for the window functions. Parameter value can be omitted for operations that do not accept a parameter.
```

Comment:
Please refer to the operation / parameter table.  

Also, I want to be able to read this and know if it is ever required. 

- For ones that don't have param, then obviously this property will be "ignored" -- not just "can be omitted"
- For ones that do have param, are they ever required, or are they always optional?

Please make this clear  (Note that Vega docs isn't very clear in this aspect too, please experiment and help jeff fix Vega docs :p)